src/hotspot/cpu/x86/macroAssembler_x86_exp.cpp

```diff
@@ -191,32 +177,31 @@ ATTRIBUTE_ALIGNED(4) juint _ONE_val[] =
 
 // Code generated by Intel C compiler for LIBM library
 
-void MacroAssembler::fast_exp(XMMRegister xmm0, XMMRegister xmm1, XMMRegister xmm2, XMMRegister xmm3, XMMRegister xmm4, XMMRegister xmm5, XMMRegister xmm6, XMMRegister xmm7, Register eax, Register ecx, Register edx, Register tmp) {
+void MacroAssembler::fast_exp(XMMRegister xmm0, XMMRegister xmm1, XMMRegister xmm2, XMMRegister xmm3,
+                              XMMRegister xmm4, XMMRegister xmm5, XMMRegister xmm6, XMMRegister xmm7,
+                              Register eax, Register ecx, Register edx, Register tmp) {
   Label L_2TAG_PACKET_0_0_2, L_2TAG_PACKET_1_0_2, L_2TAG_PACKET_2_0_2, L_2TAG_PACKET_3_0_2;
   Label L_2TAG_PACKET_4_0_2, L_2TAG_PACKET_5_0_2, L_2TAG_PACKET_6_0_2, L_2TAG_PACKET_7_0_2;
   Label L_2TAG_PACKET_8_0_2, L_2TAG_PACKET_9_0_2, L_2TAG_PACKET_10_0_2, L_2TAG_PACKET_11_0_2;
-  Label L_2TAG_PACKET_12_0_2, B1_3, B1_5, start;
+  Label L_2TAG_PACKET_12_0_2, B1_3, B1_5;
 
   assert_different_registers(tmp, eax, ecx, edx);
-  address cv = (address)_cv;
-  address Shifter = (address)_shifter;
-  address mmask = (address)_mmask;
-  address bias = (address)_bias;
+
+  address cv       = (address)_cv;
+  address mmask    = (address)_mmask;
+  address bias     = (address)_bias;
   address Tbl_addr = (address)_Tbl_addr;
-  address ALLONES = (address)_ALLONES;
-  address ebias = (address)_ebias;
-  address XMAX = (address)_XMAX;
-  address XMIN = (address)_XMIN;
-  address INF = (address)_INF;
-  address ZERO = (address)_ZERO;
-  address ONE_val = (address)_ONE_val;
-
-  bind(start);
+  address ALLONES  = (address)_ALLONES;
+  address ebias    = (address)_ebias;
+  address XMAX     = (address)_XMAX;
+  address XMIN     = (address)_XMIN;
+  address INF      = (address)_INF;
+
   subq(rsp, 24);
   movsd(Address(rsp, 8), xmm0);
   unpcklpd(xmm0, xmm0);
   movdqu(xmm1, ExternalAddress(cv));       // 0x652b82feUL, 0x40571547UL, 0x652b82feUL, 0x40571547UL
-  movdqu(xmm6, ExternalAddress(Shifter));  // 0x00000000UL, 0x43380000UL, 0x00000000UL, 0x43380000UL
+  movdqu(xmm6, ExternalAddress(SHIFTER));  // 0x00000000UL, 0x43380000UL, 0x00000000UL, 0x43380000UL
   movdqu(xmm2, ExternalAddress(16 + cv));    // 0xfefa0000UL, 0x3f862e42UL, 0xfefa0000UL, 0x3f862e42UL
   movdqu(xmm3, ExternalAddress(32 + cv));    // 0xbc9e3b3aUL, 0x3d1cf79aUL, 0xbc9e3b3aUL, 0x3d1cf79aUL
```

Comment:
If you are flipping `+` operands near the changes, maybe flip these too?

Reply:
Fixed.